### PR TITLE
helprank/data: myKeepInfos -- minor tweaks

### PR DESCRIPTION
**Pull request by raymondng**
admin ui only @yingjieMiao

### CODE REVIEW

File: kifi-backend/modules/shoebox/app/com/keepit/admin/AdminAttributionController.scala [hunk 1]
```diff
@@ -55,18 +55,30 @@ class AdminAttributionController @Inject()(
...
+      val rkr = rekeepRepo.getReKeepsByReKeeper(userId) map { k =>
+        RichReKeep(k.id, k.createdAt, k.updatedAt, k.state, userRepo.get(k.keeperId), keepRepo.get(k.keepId), uriRepo.get(k.uriId), u, keepRepo.get(k.srcKeepId), k.attributionFactor)
+      }
+      (u, rc, rk, rkr)
     }
-    Ok(html.admin.myKeepInfos(u, kc, rk))
+  }
+
+  def keepInfos(userId:Id[User]) = AdminHtmlAction.authenticated { request =>
+    val t = getKeepInfos(userId)
```
Comment (yingjieMiao) [inline kifi-backend/modules/shoebox/app/com/keepit/admin/AdminAttributionController.scala]: how about write `val (a, b, c, d) = getKeepInfos(userId)`, and replace `a, b, c, d` by meaningful names?

**Comment by yingjieMiao** [comment]
lgtm. see the style comment though.

File: kifi-backend/modules/shoebox/app/com/keepit/admin/AdminAttributionController.scala (hunk 1 above)
Comment (raymondng) [inline kifi-backend/modules/shoebox/app/com/keepit/admin/AdminAttributionController.scala]: Sounds good

**Comment by raymondng** [comment]
thanks for review!

**Pull request merged**